src/webapp/js/components/locations.js

```diff
@@ -126,17 +126,18 @@ class Locations extends Component {
     this.handleTabSelect = this.handleTabSelect.bind(this);
   }
 
+  componentWillMount() {
+    this.setState({ selectedIndex: Number.parseInt(this.props.match.params.tabIndex, 10) });
+    if (!this.props.match.params.tabIndex) {
+      this.setState({ selectedIndex: 0 });
```

Comment:
you don't need to set state to 0, it's the default value

Reply:
ok, removed